packages/web/src/pages/upload-page/UploadPage.js

```diff
@@ -262,6 +262,7 @@ class Upload extends Component {
       this.state.stems
     )
     this.changePage(Pages.FINISH)
+    this.setState({ tracks })
```

Comment:
For my own context, what happened/why was this needed?

Reply:
i dont think we need this actually.
wanted to be consistent with the state before i merged this small client PR https://github.com/AudiusProject/audius-client/pull/2042/files

but pretty sure this does not really do anything once we switch to the FINISH page.
going to simply remove and make sure things still work